TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/AbstractNodeData.php

```diff
@@ -177,7 +178,7 @@ public function setProperty($propertyName, $value)
                 return;
             }
 
-            $this->properties[$propertyName] = $value;
+            $this->properties[$propertyName] = is_string($value) ? Utility::removeControlCharactersFrom($value) : $value;
```

Comment:
I know what you mean, but this simply is a limitation of how we store strings currently. It's not even about the fusion cache so much, but about the JSON string in the database. If you have a better idea, please tell me.